skypy/halos/sampler.py

```diff
@@ -0,0 +1,75 @@
+"""Colossus halo mass sampler.
+
+This module facilitates the sampling of halos from
+`Colossus <http://www.benediktdiemer.com/code/colossus/>`_.
+
+Models
+======
+.. autosummary::
+    :nosignatures:
+    :toctree: ../api/
+
+    colossus_mass_sampler
+
+"""
+
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
```

Comment:
```suggestion
```
If colossus is not installed this could raise an error. We will import them later instead...